src/acl.c

```diff
@@ -822,7 +834,11 @@ robj *ACLDescribeUser(user *u) {
         if (selector->flags & SELECTOR_FLAG_ROOT) {
             res = sdscatfmt(res, "%s", default_perm);
         } else {
-            res = sdscatfmt(res, " (%s)", default_perm);
+            if (selectorContainsClosingParanthesis(default_perm)) {
```

Comment:
```suggestion
            /* If the selector contains a closing parenthesis, we need to quote the selector to make sure it's parsed corrrectly. */
            if (strchr(s, ')') {
```

Reply:
As discussed we would use `strnchr` if we go this route.